integration_tests/redis/redisService.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { RedisFlushModes } from '@redis/client/dist/lib/commands/FLUSHALL'
+import { createRedisClient } from '../../server/data/redisClient'
+
+export default class RedisService {
+  async deleteAll(): Promise<string> {
+    const redisClient = createRedisClient()
+    await redisClient.connect()
+    const result = await redisClient.flushAll(RedisFlushModes.ASYNC)
```

Comment:
Is ASYNC ok here - is there a chance it won't get flushed before the next test? I suspect it doesn't matyter but equally i can't see why it needs to be async?!

Reply:
good point, changed to SYNC